src/dashboard/src/requirements/base.txt

```diff
@@ -11,6 +11,7 @@ elasticsearch>=1.0.0,<2.0.0
 lazy-paged-sequence
 # Required by storage-service component
 slumber==0.6.0
+sqlparse
```

Comment:
I think I added this when I was using RunSQL instead of RunPython for the data migrations. It can probably be removed.